### PR TITLE
Add Sericea - ostree based Sway variant

### PR DESCRIPTION
**Pull request by jkonecny12**
https://fedoraproject.org/wiki/Changes/Fedora_Sway_Spin

### CODE REVIEW

File: data/profile.d/Makefile.am [hunk 1]
```diff
@@ -24,6 +24,7 @@ dist_config_DATA    = \
 	fedora-iot.conf \
 	fedora-kde.conf \
 	fedora-kinoite.conf \
+	fedora-sericea.conf \
```
Comment (VladimirSlavik) [inline data/profile.d/Makefile.am:27]: Can we just wildcard these?

**Review by VladimirSlavik — APPROVED** [review]

File: data/profile.d/Makefile.am (hunk 1 above)
Comment (jkonecny12) [reply data/profile.d/Makefile.am:27]: IIRC it is not recommended and we had a PR from outside to change place we had back than.

See here: https://github.com/rhinstaller/anaconda/pull/3619/commits/5c7ea9c2cee989d0b1b03cc7cfaf24f951271920

Comment (VladimirSlavik) [reply data/profile.d/Makefile.am:27]: Ah, sorry then - I thought that's only in srpms.

**Pull request merged**